index.html

```diff
@@ -42,7 +43,7 @@ <h1>
              </ul>
         </div>
         <section id="bellow">
-            <img src="./Images/Disabled.png"/>
+            <img src="./Images/Disabled.png" alt="Continue bellow"/>
```

Comment:
Kindly avoid overlapping images in your project😉. Probably this was introduced mistakenly while making changes and this makes your project looks a bit different from the design and can also decrease your accessibility score leading to a poor user experience.😉

<img width="682" alt="image" src="https://github.com/Arch-Noize/portfolio-microverse/assets/64643995/92fbb134-9d1e-456c-a65d-25a4d7c68494">

Reply:
Hi @Ibizugbe, 

Thanks for the review. I am not sure how that image got there... I went through different screen sizes and I don't see it display that way. Can you share the resolution you used, please?